### DISCUSSION TITLE
Welcome to Itchy for Android Discussions!

### DISCUSSION THREAD

**Discussion by micahlt** [Scratch-Client-4/itchy-cordova#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions related to Itchy
  * Share ideas for future features
  * Engage with other community members and developers
  * Welcome others and are open-minded. Remember that this is a community we
  build together!

  To get started, comment below with an introduction of yourself and tell us about your experience with Itchy!

**Comment by retronbv** [comment]
When is itchy gonna be released? Also what happened to the iOS version? I can't find it anywere

**Reply by micahlt** [reply]
Itchy has been pushed to mid-2021 due to my life being busy.  However, there never was an iOS version.

**Reply by retronbv** [reply]
Oh, I thought Incro was doing it.

**Reply by micahlt** [reply]
Technically that's not Itchy, but I do think he's given up on that project 😢

**Reply by retronbv** [reply]
oh ok :c
